src/index.test.ts

```diff
@@ -287,7 +287,7 @@ describe(`wallet addresses to names`, () => {
     });
 
     test(`wallets with no names return null`, async () => {
-      const result = await walletToDotAbcDotBonkOrDotPoor(
+      const result = await walletToDotAnything(
         connection,
         WALLET_WITH_NO_NAMES
       );
```

Comment:
Can you please add a new test for one of the wildcard domains? Like `a wallet address with an arbitrary domain registered at ANS returns the wallet name`?

Reply:
can you give me more details on this, I don't quite understand what is needed

Reply:
Handled in our call